chainermn/datasets/empty_dataset.py

```diff
@@ -0,0 +1,21 @@
+import chainer
+
+
+def create_empty_dataset(dataset):
+    """Creates an empty dataset for models with no inputs and outputs.
+
+    This function generates an empty dataset, i.e., ``__getitem__()`` only
+    returns ``None``. Its dataset is compatible with the original one.
+    Such datasets used for models which do not take any inputs,
+    neither return any outputs. We expect models, e.g., whose ``forward()``
+    is starting with ``chainermn.functions.recv()`` and ending with
+    ``chainermn.functions.send()``.
+
+    Args:
+        dataset(chainer.datasets.TupleDataset): Dataset to convert.
```

Comment:
`dataset` does not need to be `TupleDataset`. Chainer accepts many kinds of datasets.